src/AbstractPhpdocToTypeDeclarationFixer.php

```diff
@@ -31,7 +31,7 @@
  */
 abstract class AbstractPhpdocToTypeDeclarationFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
-    private const CLASS_REGEX = '/^\\\\?[a-zA-Z_\\x7f-\\xff](?:\\\\?[a-zA-Z0-9_\\x7f-\\xff]+)*$/';
+    private const CLASS_REGEX = '/^\\\\?[A-Z_a-z\x80-\xff](?:\\\\?[\dA-Z_a-z\x80-\xff]+)*$/';
```

Comment:
Please do not make changes like from `a-zA-Z0-9_` to `\dA-Z_a-z` because it's exactly the same, and since it does not provide any actual change it's unnecessary to review such diffs. Please bring back previous block in each modified regex, so we can see actual change in the diff.

```suggestion
    private const CLASS_REGEX = '/^\\\\?[a-zA-Z_\x80-\xff](?:\\\\?[a-zA-Z0-9_\x80-\xff]+)*$/';
```

Reply:
This unifies the class regex to be ASCII ordered and consistent across the whole project. Is that ok?

Reply:
Personally I find previous order better and more readable, and I don't like mixing such changes - SRP works in PR scope too 😉. @kubawerlos what do you think?

Reply:
Regex was reverted only in 1 file, but such change was made in several files :wink:.

Reply:
I have reverted it, previously it was inconsistent and `\x7f` was wrongly present. I will move the identifier regexes into one place in another PR later.

Reply:
Will be improved in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7010, let's merge this PR as is.

Reply:
@Wirone can you please have a look on this PR? I can then finish #7010, it will remove these fishy chars groups completely. And there are even wrong, as they do not work with `u` modifier (fixed in #7010, not to be done in this PR).

Reply:
@Wirone I have reverted all unrelated changed. Now this PR is pure fix for #6959.